scripts/model/blog.js

```diff
@@ -42,27 +54,24 @@
         url: url,
```

Comment:
Very nice. I like how the url/data source is abstracted since you are pulling in two separate data files.